app/src/main/java/com/marverenic/music/player/ServicePlayerController.java

```diff
@@ -342,6 +347,60 @@ public void setQueue(List<Song> newQueue, int newPosition) {
         });
     }
 
+    /**
+     * This function works like {@link #setQueue(List, int)}, except it will split queue to chunks
+     * and send one by one (keep order) to remote service. if newQueue size is
+     * less than {@link #MINIMUM_SONG_PER_CHUNK}, use {@link #setQueue(List, int)} instead.
+     * Needn't expose this method.
+     *
+     * @param newQueue New queue which contains large number of songs.
+     * @param newPosition The index in the queue to start playback from
+     */
+    private void sendBigQueue(List<Song> newQueue, int newPosition, int flag) {
+        execute(() -> {
+            try {
+                mBinding.beginBigQueue();
+                int i;
```

Comment:
This is super nitpicky, but just for consistency with the `getBigQueue()` method, I think that this variable should be renamed from `i` to `offset`.